DoSomething/intertwine
# add Bully Text 2015 game, filename tweaks

#### What's this PR do?

Adds Bully Text 2015 to the preloads, so that the playable file whenever someone navigates to `dosomething.github.io/intertwine`. Also made some name changes to the other preload files, for consistency and professionalism. 
#### How should this be manually tested?

Run `grunt`, start up a local server in the intertwine directory with `python -m SimpleHTTPServer 8080`, test to see if the game exists and you can create it. 
#### What are the relevant tickets?

Closes #54.
